tests/dash/app_clustergram.py

```diff
@@ -159,9 +159,9 @@ def layout():
                 html.Hr(),
 
                 html.Div(
-                    'Header of row labels column in uploaded dataset',
+                    'Header of index column in uploaded dataset',
                     title='If a dataset was uploaded, enter the header of' +
-                    'the column that contains the title of each row.',
+                    'the column that contains the indices.',
```

Comment:
Oh, okay, now I understand the meaning! I would therefore suggest the more conventional phrasing "enter the name of the column to use as index."